manifests/base/rbac/role.yaml

```diff
@@ -51,14 +51,14 @@ rules:
   - trainer.kubeflow.org
   resources:
   - clustertrainingruntimes
-  - trainingruntimes
   verbs:
   - get
   - list
   - watch
 - apiGroups:
   - trainer.kubeflow.org
   resources:
+  - trainingruntimes
```

Comment:
Practically the operator should not be granted permission to create or delete TrainingRuntimes.

Reply:
I guess, patch will be enough, right @tenzen-y ?

Reply:
Good point. I will remove unneeded permissions.

Reply:
Done.